hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/handlers/ThroughputLimiter.java

```diff
@@ -0,0 +1,26 @@
+package pl.allegro.tech.hermes.frontend.publishing.handlers;
+
+import com.codahale.metrics.Metered;
+import pl.allegro.tech.hermes.api.TopicName;
+
+import static java.lang.String.format;
+
+public interface ThroughputLimiter {
+    void check(TopicName topic, Metered throughput);
```

Comment:
this interfaces contract is hidden behind assumptions about throwing exceptions of proper kinds which are used as flow control - please return values